interm_notebook_5_classify.ipynb

```diff
@@ -0,0 +1,1175 @@
+{
```

Comment:
<p>This is fine, some level of the "author's voice" is expected to come through in the prose</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>